src/admin/apiProvider/dataProviders/siteDataProvider.ts

```diff
@@ -1,50 +1,36 @@
-import { DataProvider } from "react-admin";
+import { DataProvider, HttpError } from "react-admin";
 
+import { loadFullSite, loadSiteIndex } from "@/connections/Entity";
 import {
   DeleteV2AdminSitesUUIDError,
   fetchDeleteV2AdminSitesUUID,
-  fetchGetV2AdminSites,
   fetchGetV2AdminSitesMulti,
-  fetchGetV2ENTITYUUID,
-  GetV2AdminSitesError,
-  GetV2AdminSitesMultiError,
-  GetV2ENTITYUUIDError
+  GetV2AdminSitesMultiError
 } from "@/generated/apiComponents";
 
 import { getFormattedErrorForRA } from "../utils/error";
-import { apiListResponseToRAListResult, raListParamsToQueryParams } from "../utils/listing";
-
-const siteSortableList = ["name", "project_name", "start_date"];
+import { entitiesListResult, raConnectionProps } from "../utils/listing";
 
 // @ts-ignore
 export const siteDataProvider: DataProvider = {
+  // @ts-expect-error until we can get the whole DataProvider on Project DTOs
```

Comment:
`Site DTOs` instead of `Project` for this copied comment.

Reply:
done